src/upgrades/mod.rs

```diff
@@ -40,6 +98,8 @@ pub enum UpgradeError {
 
 impl fmt::Display for UpgradeError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Upgrade failed. Check your data directory. If you find folders named 'arch-cas-[$version]' or 'arch-pubd-[$version]' there, then rename them to 'cas' and 'pubd' respectively and re-install your old $version of krill. Please contact us and let us know about this error. You can reach us at rpki-team@nlnetlabs.nl. Detailed error: ")?;
```

Comment:
Could/should this rename be handled automatically?

Reply:
Improved the error reporting. It will only talk about this now if the finalize step failed. We cannot recover from that failure so leaving it to the user. It is very unlikely to fail though because essentially we are then just renaming dirs, u updating version files and cleaning up. I/o issues would have come up before this step.